src/iOS/Xamarin.iOS/Samples/Symbology/FeatureLayerExtrusion/FeatureLayerExtrusion.cs

```diff
@@ -7,15 +7,16 @@
 // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
 // language governing permissions and limitations under the License.
 
+using System;
+using System.Drawing;
+using CoreGraphics;
 using Esri.ArcGISRuntime.Data;
 using Esri.ArcGISRuntime.Geometry;
 using Esri.ArcGISRuntime.Mapping;
 using Esri.ArcGISRuntime.Symbology;
 using Esri.ArcGISRuntime.UI;
 using Esri.ArcGISRuntime.UI.Controls;
 using Foundation;
-using System;
-using System.Drawing;
 using UIKit;
 
 namespace ArcGISRuntime.Samples.FeatureLayerExtrusion
```

Comment:
Note for the future csproj-fixing effort: **FeatureLayerExtrusion.jpg** is listed twice in the project file. That's the last of them.